### PR TITLE
chore: even more doc work

### PR DESCRIPTION
**Pull request by ahammond**
Fixes #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,3 +66,36 @@ Watch with awe and wonder as projen stamps out a project with
...
+
+```bash
+cat <<EOF >>~/.npmrc
+
+//npm.pkg.github.com/:_authToken=ghp_your-github-token
+@time-loop:registry=https://npm.pkg.github.com/
+EOF
+```
+
+NOTE: There are also some `@clickup/*` libraries in ye olde `npmjs.com`.
```
Comment (Justinon) [inline README.md:93]: 🤠

File: README.md [hunk 2]
```diff
@@ -70,3 +66,36 @@ Watch with awe and wonder as projen stamps out a project with
...
+
+### Add It To Your App
+
+Now that the GitHub package repo is configured, you can just
+
+```
+yarn add @time-loop/my-new-lib --save
```
Comment (Justinon) [inline README.md:100]:
```suggestion
yarn add @time-loop/my-new-cdk-app --save
```

File: README.md [hunk 3]
```diff
@@ -43,7 +39,7 @@ Watch with awe and wonder as projen stamps out a project with
 - Standard prettier and eslint configs.
 - For CDK apps, CDKv2 support with `@time-loop/cdk-library`, `cdk-constants`, `cdk-iam-floyd`, `colors` and `multi-convention-namer` libraries pre-installed for your coding pleasure.
 
-## What's Left For You To DO
+## What's Left For You To Do
 
 - Create the repo in GitHub. `gh repo create time-loop/my-new-lib --private --push --source=.` will probably do what you need. Note: the names MUST MATCH.
```
Comment (Justinon) [inline README.md:44]:
```suggestion
- Create the repo in GitHub. `gh repo create time-loop/my-new-cdk-app --private --push --source=.` will probably do what you need. Note: the names MUST MATCH.
```

File: README.md [hunk 4]
```diff
@@ -13,21 +13,17 @@
...
 
 ### ClickUpTypeScriptProject
 
 When creating new TypeScript Libraries:
 
 ```bash
-yarn global add @time-loop/clickup-projen
-
 mkdir my-new-lib
 cd my-new-lib
```
Comment (Justinon) [inline README.md:28]:
```suggestion
mkdir my-new-cdk-app
cd my-new-cdk-app
```

File: README.md (hunk 3 above)
Comment (Justinon) [reply README.md:44]: you get the idea. Do this everywhere.

**Review by Justinon — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (ahammond) [reply README.md:100]: Nope. I don't have an example of a cdk lib (or, for that matter, support for them outside of `cdk-library`). We might want to create some open-source constructs in the future which would drive this, but for now, we don't have this.

File: README.md (hunk 3 above)
Comment (ahammond) [reply README.md:44]: No. While we have support in here for cdk-apps, it's not my primary focus. We'll probably pound out a few dozen cdk apps. `clickup_frontend` alone has over 300 libraries. So... that's my target audience here.

File: README.md (hunk 4 above)
Comment (ahammond) [reply README.md:28]: Nope. This is explicitly an example of how to create a TypeScript Project. There's an example of a CdkTypeScriptApp above.

**Pull request merged**